manifests/config.pp

```diff
@@ -6,9 +6,10 @@
   }
 
   concat { $dns::publicviewpath:
-    owner => root,
-    group => $dns::params::group,
-    mode  => '0640',
+    owner        => root,
+    group        => $dns::params::group,
+    mode         => '0640',
+    validate_cmd => "${dns::params::named_checkconf} %",
```

Comment:
Since the `dns` class inherits from `dns::params`, you can use the following which allows making it a parameter as well.

```suggestion
    validate_cmd => "${dns::named_checkconf} %",
```

Reply:
My initial thought was to handle `named_checkconf` in a similar way to `rndcconfgen`, which is not exposed as a class parameter explicitly. But I will implement your suggestion at once :)